### PR TITLE
Added Colors

### PR DESCRIPTION
**Pull request by florianschulte**
Added Colors to the Project

### CODE REVIEW

**Comment by alexandrepiveteau** [comment]
@florianschulte Would it eventually be possible to add some prefixes to the color names (for instance the `quadro_` prefix ) ?

This would make sure we do not mistakenly use a non-Quadro color offered by the support libraries or another tool we'd be using.

**Comment by florianschulte** [comment]
Sure, makes sense I will add that to all our Icons and Values

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -26,63 +26,63 @@
...
+    <color name="quadro_sauerland_20">#5ED700</color>
+    <color name="quadro_sauerland_40">#00CB0D</color>
+    <color name="quadro_sauerland_60">#00BE41</color>
 
     <!-- Text colors -->
-    <color name="light_primary">#FFFFFF</color>
-    <color name="light_secondary">#B3FFFFFF</color>
-    <color name="light_tertiary">#80FFFFFF</color>
-    <color name="light_divider">#1FFFFFFF</color>
+    <color name="quadro_ight_primary">#FFFFFF</color>
```
Comment (alexandrepiveteau) [inline app/src/main/res/values/colors.xml]: @florianschulte Small typo here 😄

Comment (florianschulte) [reply app/src/main/res/values/colors.xml]: oops

**Pull request merged**